dali/test/python/test_fw_iterators.py

```diff
@@ -149,6 +150,43 @@ def test_mxnet_iterator_last_batch_no_pad_last_batch():
     assert len(img_ids_list_set) == data_size
     assert len(set(mirrored_data)) != 1
 
+def test_mxnet_iterator_empty_array():
+    from nvidia.dali.plugin.mxnet import DALIGenericIterator as MXNetIterator
+
+    batch_size = 4
+    size = 100
```

Comment:
I'm not sure if we need that many, iteration to iteration data is exactly the same.

Reply:
Done